internal/kconfig/kconfig.go

```
@@ -58,7 +58,7 @@ func Parse(source io.ReaderAt) (map[string]string, error) {
 	for s.Scan() {
```

We could bail out of this loop once we've found all the filteredConfigs. Upside is much faster parsing if the configs are towards the beginning of the file, downside is that perf is a lot less predicable and we might not discover some parsing error. What do you think?

```suggestion
	for s.Scan() && (filteredConfigs != nil && len(ret) != len(filteredConfigs)) {
```